src/proof/tree_assets.rs

```diff
@@ -64,7 +64,7 @@ where
     }
 
     fn pos(&self) -> u64 {
-        self.0.pos()
+        *self.0.pos()
```

Comment:
I understand that if we support also `wasm32` it might have sense returns a reference to a `u64`, but then we should do the same here as well no?

Reply:
This is a trait-method. Therefore I'm forced to return by value and therefore, copy.

Reply:
Should be solved now with the latest rc release of `dusk-poseidon`